starter_code/app.js

```diff
@@ -1,18 +1,59 @@
+const express = require("express");
+const hbs = require("hbs");
+
+const app = express();
+
+app.use (express.static(__dirname + "/public"));
+
+app.set("views", __dirname + "/views");
+//what is "view engine" ?
+app.set("view engine", "hbs");
+
+app.set("layout",__dirname + "/views/layout.hbs");
+
+//what is partials?
```

Comment:
partials are just pieces of view code we might want to reuse in our app, instead of rewriting the markup